lib/shanty_docker_plugin.rb

```diff
@@ -0,0 +1,6 @@
+# Public: shanty docker plugin
```

Comment:
I think you want to require all the other things you need from here so that when somebody does `require 'shanty_docker_plugin', it works as expected.